src/components/Select/Select.module.scss

```diff
@@ -0,0 +1,255 @@
+@import '../../styles/mixins';
+
+.select-input {
+  display: inline-block;
+  :global {
+    .select-input__label {
```

Comment:
These classes are defined by ourself so they don't need to be in the global context.
Use the global context only when overriding classes from external libraries.